chia/wallet/wallet_state_manager.py

```diff
@@ -1474,32 +1474,30 @@ async def coin_added(
             wallet_id,
             wallet_type,
         )
-        farmer_reward = False
-        pool_reward = False
-        if self.is_farmer_reward(height, coin):
-            farmer_reward = True
-        elif self.is_pool_reward(height, coin):
-            pool_reward = True
-
-        farm_reward = False
-        parent_coin_record: Optional[WalletCoinRecord] = await self.coin_store.get_coin_record(coin.parent_coin_info)
-        if parent_coin_record is not None and wallet_type.value == parent_coin_record.wallet_type:
-            change = True
+
+        if self.is_pool_reward(height, coin):
+            tx_type = TransactionType.COINBASE_REWARD
```

Comment:
is `tx_type` an `int`? it was prior to this patch. I think keeping the type annotation would be a good idea.
Also, previously we ask the enum constants for their `.value`, you dropped that, which may have different behavior.
I imagine restoring the type annotation might highlight this.

Reply:
Its an `IntEnum` yes.